test/fixtures/cli-plugin-array/.gitattributes

```diff
@@ -0,0 +1 @@
+*.html text=auto eol=lf
```

Comment:
Ah, okay. Yeah, that's totally fine. We're testing Metalsmith here, not the `metalsmith-markdown` plugin. Plus I'm not sure that always outputting html files with Unix-style newlines is even a bad idea...